text/0083-Improve-the-robust-of-balance-scheduler.md

```diff
@@ -0,0 +1,65 @@
+# Improve the robust of balance scheduler
```

Comment:
It seems this RFC only talks about the balance region scheduler. Maybe you need to change the title.